students/bcoates/lesson06/mailroom4.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+# Starting list of donors and their donation amounts
+donor_history = {
+    "donor_1": {"name": "Lana Kane", "donations": [2999.99]},
+    "donor_2": {"name": "Cheryl Tunt", "donations": [150.20, 98192.10]},
+    "donor_3": {"name": "Cyril Figgis", "donations": [819.25, 998.62, 10.50]},
+    "donor_4": {"name": "Pam Poovey", "donations": [74926.10, 2675.87, 3289.33]},
+    "donor_5": {"name": "Ray Gillette", "donations": [3820.90]}
```

Comment:
this isn't ideal structure for this program - your key should benefit you but it does not in this case.

If you needed to find a value in your dictionary, how would you know what key to use? You program uses donor names to update your data in which case you should use donor name as a key.